<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Place</class>
 <widget class="QWidget" name="Place">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVelo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>velo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNbVelo">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>130</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nb</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
